### PR TITLE
DOC: Add references to NA NEP.

### PR DESCRIPTION
**Pull request by FelixBenning**
Added new Information to the NEP 26 (Summary of the debate of sentinel values vs masks)

### CODE REVIEW

**Comment by mattip** [comment]
If there is no performance advantage to bitmasks or sentinels, wouldn't it be better to state that rather than to make a claim and then retract it in the next line?

**Comment by FelixBenning** [comment]
I guess, just did not know how to put it concisely.

File: doc/neps/nep-0026-missing-data-summary.rst [hunk 1]
```diff
@@ -717,6 +717,16 @@ implementation strategy for NA dtypes.
...
+Additional References (2020):
+--------------------
+
+In 2018 Wes McKinney demonstrates, that sentinel values are actually less performant than bitmasks (contrary to popular belief also expressed here):
+https://wesmckinney.com/blog/bitmaps-vs-sentinel-values/
+
+In a follow-up Pasha Stetsenko optimizes the code for sentinel values by Wes McKinney, allowing them to perform similarly (but not better):
+https://github.com/st-pasha/microbench-nas/blob/master/README.md
+
+Note that the bitmasks used by Wes McKinnely are tightly packed and can not store additional information like "degree of missingness" mentioned as possible advantages of masks. For more details check out the Apache Arrow project, where this method is used.
```
Comment (rossbar) [inline doc/neps/nep-0026-missing-data-summary.rst:729]: I'm not sure that this statement is appropriate in the "Additional References" section. Perhaps just a reference to Apache Arrow (preferably to the specific documentation related to bitmasks) would be more appropriate?

File: doc/neps/nep-0026-missing-data-summary.rst [hunk 2]
```diff
@@ -5,7 +5,7 @@ NEP 26 — Summary of Missing Data NEPs and discussion
 :Author: Mark Wiebe <mwwiebe@gmail.com>, Nathaniel J. Smith <njs@pobox.com>
 :Status: Deferred
 :Type: Standards Track
-:Created: 2012-04-22
+:Created: 2012-04-22 (added references: 2020-04-06)
```
Comment (rossbar) [inline doc/neps/nep-0026-missing-data-summary.rst:8]:
```suggestion
:Created: 2012-04-22
```
For consistency with the other NEPS, I think this should be removed. There is an `:Updated:` field that may be more appropriate.

File: doc/neps/nep-0026-missing-data-summary.rst [hunk 3]
```diff
@@ -717,6 +717,16 @@ implementation strategy for NA dtypes.
...
 https://github.com/njsmith/numpy/wiki/NA-discussion-status
 
+Additional References (2020):
+--------------------
+
+In 2018 Wes McKinney demonstrates, that sentinel values are actually less performant than bitmasks (contrary to popular belief also expressed here):
+https://wesmckinney.com/blog/bitmaps-vs-sentinel-values/
+
+In a follow-up Pasha Stetsenko optimizes the code for sentinel values by Wes McKinney, allowing them to perform similarly (but not better):
+https://github.com/st-pasha/microbench-nas/blob/master/README.md
```
Comment (rossbar) [inline doc/neps/nep-0026-missing-data-summary.rst:727]: This seems more like discussion than a reference, and agree that it sort of undermines the statement above the first reference and is thus confusing.

**Review by rossbar — COMMENTED** [review]
I don't know what the policy is regarding adding new information to deferred NEPs, i.e. whether documents with this status are intended more for archival purposes or for spurring discussion. 

The blog post was interesting and generally fits with the topic of the NEP, though I found it difficult to connect the referenced material back to any specific part of the NEP as it's provided without any link to or context from specific discussion in the NEP itself. I feel this makes it of limited use to readers, though perhaps valuable from an archival perspective.

**Comment by seberg** [comment]
I am going to close this for now. We discussed it briefly. We may need to add a new discussion about this at some point. And at that point we should link it, but right now I am not sure this adds much.

Thanks for the effort though! The NA things are an important fact, but it feels like we would need to add a lot more context to make it helpful. At which point we may need a new documentation somewhere and then we could link that from a top banner.

**Pull request closed without merging**